test/e2e/proxy_publicviewer_test.go

```diff
@@ -0,0 +1,210 @@
+			token, err := authsupport.NewTokenFromIdentity(userIdentity, claims...)
+			require.NoError(t, err)
+
+			joeCli, err := hostAwait.CreateAPIProxyClient(t, token, hostAwait.ProxyURLWithWorkspaceContext(sp.Name))
+			require.NoError(t, err)
+
+			t.Run("sso user can list config maps from space", func(t *testing.T) {
+				// then
+				cms := corev1.ConfigMapList{}
+				err := joeCli.List(context.TODO(), &cms, client.InNamespace(sp.Status.ProvisionedNamespaces[0].Name))
+				require.NoError(t, err)
+			})
+
+			t.Run("sso user cannot create config maps into space", func(t *testing.T) {
+				// then
+				cm := corev1.ConfigMap{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "test-cm",
+						Namespace: sp.Status.ProvisionedNamespaces[0].Name,
+					},
+				}
+				err := joeCli.Create(context.TODO(), &cm)
+				require.True(t, errors.IsForbidden(err), "expected Create ConfigMap as SSO user to return a Forbidden error, actual: %v", err)
+			})
+		})
+	})
+
+	t.Run("space is not flagged as community", func(t *testing.T) {
+		sp := toolchainv1alpha1.Space{}
+		err = hostAwait.Client.Get(context.TODO(), client.ObjectKeyFromObject(space), &sp)
+		require.NoError(t, err)
+
+		/*
+		   Given Space exists for user A
+		   Given User community-user exists
+		   Given A space's visibility is NOT "community"
+		   Then  community-user cannot view A's Space
+		   And   community-user cannot create resources in A's Space
+		*/
+		t.Run("community user cannot access to non-community space", func(t *testing.T) {
+			require.NotEmpty(t, sp.Status.ProvisionedNamespaces)
+
+			t.Run("community user cannot list config maps from community space", func(t *testing.T) {
+				// then
+				cms := corev1.ConfigMapList{}
+
+				communityUserProxyClient, err := hostAwait.CreateAPIProxyClient(t, communityUser.token, hostAwait.ProxyURLWithWorkspaceContext(sp.Name))
+				require.NoError(t, err)
+
+				err = communityUserProxyClient.List(context.TODO(), &cms, client.InNamespace(sp.Status.ProvisionedNamespaces[0].Name))
+				require.True(t, errors.IsForbidden(err), "expected Create ConfigMap as community user to return a Forbidden error, actual: %v", err)
+			})
+
+			t.Run("community user cannot create config maps into space", func(t *testing.T) {
+				cm := corev1.ConfigMap{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      "test-cm",
+						Namespace: sp.Status.ProvisionedNamespaces[0].Name,
+					},
+				}
+				err := communityUserProxyClient.Create(context.TODO(), &cm)
+				require.True(t, errors.IsForbidden(err), "expected Create ConfigMap as community user to return a Forbidden error, actual: %v", err)
+			})
+		})
+	})
+}
+
+func createAppStudioUser(t *testing.T, awaitilities wait.Awaitilities, user *proxyUser) {
+	// Create and approve signup
+	req := NewSignupRequest(awaitilities).
+		Username(user.username).
+		IdentityID(user.identityID).
+		ManuallyApprove().
+		TargetCluster(user.expectedMemberCluster).
+		EnsureMUR().
```

Comment:
minor optimization - If I'm not mistaken you don't need a space to be created for this user right ? . 
```suggestion
		EnsureMUR().
		NoSpace().
```

Reply:
done in c00f15122abb7185057104da91dd426921dc3255